hfc.py

```diff
@@ -32,33 +32,36 @@ def __str__(self):
     gh_name = ""
     while gh_name == "":
         gh_name = input("Please enter your github username: ")
-    #Make request to github api
-    req = requests.get("https://api.github.com/search/issues?q=author:{}%20type:pr%20created:%3E2017-09-30%20created:%3C2017-11-01".format(gh_name))
-    #Get a json from the request
-    json_out = req.json()
-    if "errors" in json_out:
-        print("The username you typed does not exist or you don't have access to view this user's profile.")
-        exit(1)
-    prs = []
-    pr_count = json_out["total_count"]
+    try:
+        #Make request to github api
+        req = requests.get("https://api.github.com/search/issues?q=author:{}%20type:pr%20created:%3E2017-09-30%20created:%3C2017-11-01".format(gh_name))
```

Comment:
I think it would be better if only this line gets the handling. For example:
```
try:
    req = requests.get("https://api.github.com/search/issues?q=author:{}%20type:pr%20created:%3E2017-09-30%20created:%3C2017-11-01".format(gh_name))
except requests.RequestException:
    print('\nERROR CONNECTING TO THE INTERNET\n')
    exit(1)
```